tests/telemetry_derived/clients_daily_v7/test_first_value_falsey/expect.ndjson

```diff
@@ -1 +0,0 @@
-{"active_addons":{"list":[]},"addon_compatibility_check_enabled":false,"app_build_id":"","app_display_version":"","app_name":"","app_version":"","attribution":{"campaign":"","content":"","medium":"","source":""},"blocklist_enabled":false,"channel":"","client_id":"test","cpu_cores":0,"cpu_count":0,"cpu_family":0,"cpu_l2_cache_kb":0,"cpu_l3_cache_kb":0,"cpu_model":0,"cpu_speed_mhz":0,"cpu_stepping":0,"cpu_vendor":"","default_search_engine":"","default_search_engine_data_load_path":"","default_search_engine_data_name":"","default_search_engine_data_origin":"","default_search_engine_data_submission_url":"","distribution_id":"","e10s_enabled":false,"env_build_arch":"","env_build_id":"","env_build_version":"","experiments":{"key_value":[]},"flash_version":"","gfx_features_advanced_layers_status":"","gfx_features_d2d_status":"","gfx_features_d3d11_status":"","gfx_features_gpu_process_status":"","install_year":0,"is_default_browser":false,"is_wow64":false,"locale":"","memory_mb":0,"normalized_channel":"","normalized_os_version":"","os":"","os_service_pack_major":0,"os_service_pack_minor":0,"os_version":"","pings_aggregated_by_this_row":3,"previous_build_id":"","sample_id":"test","sandbox_effective_content_process_level":0,"scalar_parent_aushelper_websense_reg_version":"","scalar_parent_devtools_accessibility_select_accessible_for_node_sum":{"key_value":[]},"search_cohort":"","sessions_started_on_this_day":0,"submission_date":"2019-01-01","sync_configured":false,"telemetry_enabled":false,"timezone_offset":0,"update_auto_download":false,"update_channel":"","update_enabled":false,"vendor":"","windows_build_number":0,"windows_ubr":0}
```

Comment:
deleted this test because `FIRST_VALUE` is no longer used, and `udf_mode_last` is tested separately